bin/dredd

```diff
@@ -33,18 +33,38 @@ if (argv._[1] === undefined) {
   console.error("Error: Must specify api endpoint.");
   argError = true;
 }
+
 if (argError) {
   console.error("\n");
   optimist.showHelp(fn=console.error);
   process.exit(1);
 }
 
+// transform path and p argument to array if it's not
+if(!Array.isArray(argv['path'])){
```

Comment:
There are methods in `applyConfiguration` to take care of this, see: https://github.com/apiaryio/dredd/blob/master/src/apply-configuration.coffee#L43

The transformation could be removed from here add two lines added in apply-configuration:
```javascript
 configuration.options.path = coerceToArray(configuration.options.path)
 configuration.options.only = coerceToArray(configuration.options.only)
```

Unless I'm misunderstanding and there's a reason they need to be parsed before the rest of the options?